### DISCUSSION TITLE
有什么能打steam游戏的代理推荐吗？

### DISCUSSION THREAD

**Discussion by YdayangY** [v2fly/v2ray-core#2421] [Q&A]
如题，想进vrchat学英语 但是加速器都有点慢还不稳定，有代理的话可以推荐一下谢谢。

**Comment by mydogshitgold** [comment]
I failed to see how this is related to V2Ray. 

And [Steam itself is not happy with proxy](https://store.steampowered.com/subscriber_agreement/)

> You agree that you will not use IP proxying or other methods to disguise the place of your residence, whether to circumvent geographical restrictions on game content, to order or purchase at pricing not applicable to your geography, or for any other purpose. If you do this, Valve may terminate your access to your Account.

If you have any questions about this, ask [GabeN](mailto:gaben@valvesoftware.com) himself.

**Reply by maliohammad** [reply]
Steam is against it but they are not strict with banning over the use of vpn/ proxies. 
I think the user was asking about which protocol would give the best ping and the least packet loss.

**Reply by mydogshitgold** [reply]
If the absolute latency remains high, the question of whether any protocols can alleviate the problem is answered in the negative. It is therefore optimal to opt for servers  geographically close to the end user, to use [multiplexing](https://guide.v2fly.org/en_US/advanced/mux.html) and, where plausible, [to avoid using TCP-based protocols](https://www.youtube.com/watch?v=AAssk2N_oPk).
